roles/update_containers/README.md

```diff
@@ -7,6 +7,7 @@ openstack services containers, Ansibleee and EDPM Baremetal Image.
 If apply, please explain the privilege escalation done in this role.
 
 ## Parameters
+* `cifmw_update_containers`: The boolean value which will decide to run the 'Update the containers' task. Default to `false`.
```

Comment:
Role, not task. Although technically is the 'main' task, while the other two tasks in the role just create the file, that this task will use... the logic is also a little bit messed, I would put all the tasks in one block with a when cifmw_update_containers is set to true, but 🤷